stats/stats.go

```diff
@@ -539,3 +539,31 @@ func RoundToDigits(v float64, digits int) float64 {
 func Round(v float64) float64 {
 	return RoundToDigits(v, 4)
 }
+
+// Map is a type that stores the key value pair which the key is string and value is int.
+type Map struct {
```

Comment:
Map is quite a vague/generic name here (even in stats.)

maybe it's `Occurrences`  or some such

Reply:
updated